openquake/calculators/extract.py

```diff
@@ -793,8 +799,10 @@ def extract_agg_curves(dstore, what):
     if qdic['absolute'] == [1]:
         pass
     elif qdic['absolute'] == [0]:
-        evalue, = dstore['agg_values'][agg_id][lts]
-        arr /= evalue
+        evalue_sum = 0
+        for lt in lts[0].split('+'):
```

Comment:
It does not feel right; are we sure that the list `lts` has only a single element in all situations? If not, do we want to force that?

Reply:
I added a loop to make sure all the elements in `lts` are taken into accoun